bes_theme/assets/data/OSSP-legend.js

```diff
@@ -0,0 +1,567 @@
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-A",
+                "name": "TD-C-A",
+                "color": "B014E9",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  API clients"
+            },
+            "TD-C-D" :   {
+                "id": 3374498169,
+                "node_id": "LA_kwDOFx7Zis7JIsF5",
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-D",
+                "name": "TD-C-D",
+                "color": "22EB7C",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Database"
+            },
+
+            "TD-C-DA" :   {
+                "id": 3374499627,
+                "node_id": "LA_kwDOFx7Zis7JIscr",
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-DA",
+                "name": "TD-C-DA",
+                "color": "7AF9A2",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Distributed Application"
+            },
+            "TD-C-Ms" :   {
+                "id": 3374500531,
+                "node_id": "LA_kwDOFx7Zis7JIsqz",
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-Ms",
+                "name": "TD-C-Ms",
+                "color": "034A51",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Microservices"
+            },
+
+            "TD-C-C" :   {
+                "id": 3374501108,
+                "node_id": "LA_kwDOFx7Zis7JIsz0",
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-C",
+                "name": "TD-C-C",
+                "color": "284F6B",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Container"
+            },
+
+            "TD-C-CA" :   {
+                "id": 3374488303,
+                "node_id": "LA_kwDOFx7Zis7JIprv",
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-CA",
+                "name": "TD-C-CA",
+                "color": "0052cc",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Client Application"
+            },
+
+            "TD-C-Wlt" :   {
+                "id": 3374502074,
+                "node_id": "LA_kwDOFx7Zis7JItC6",
+                "url": "https://api.github.com/repos/Be-Secure/BeSLighthouse/labels/TD-C-Wlt",
+                "name": "TD-C-Wlt",
+                "color": "749812",
+                "default": false,
+                "description": "Technology  Domain  Classifier: Composition:  Type :  Wallet App"
+            }
+        },
+
+        "COM": {
+            "COM - C" : {
```

Comment:
@asa1997 Each foundation must be a separate label , the structure should change to reflect the same

Reply:
At the moment, foundation labels are not available.
Will be raising PR  updated with newly created foundation based labels.